Bigtable/src/Connection/Grpc.php

```diff
@@ -325,15 +337,67 @@ public function getTable(array $args)
      */
     public function deleteTable(array $args)
     {
-        throw new \BadMethodCallException('This method is not implemented yet');
+        $name = $this->pluck('name', $args);
+        return $this->send([$this->bigtableTableAdminClient, 'deleteTable'], [
+            $name,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
     }
 
     /**
      * @param array $args
      */
     public function modifyColumnFamilies(array $args)
     {
-        throw new \BadMethodCallException('This method is not implemented yet');
+        $name = $this->pluck('name', $args);
+        $modifications = $this->pluck('modifications', $args);
+        return $this->send([$this->bigtableTableAdminClient, 'modifyColumnFamilies'], [
+            $name,
+            array_map([$this,'columnFamilyModificationsObject'], $modifications),
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $modifications
+     * @return Modification
+     */
+    private function columnFamilyModificationsObject(array $modifications)
+    {
+
+        if (isset($modifications['create'])) {
+            $create = $this->pluck('create', $modifications);
+            $modifications['create'] = $this->columnFamilyObject($create);
+        } elseif (isset($modifications['update'])) {
+            $update = $this->pluck('update', $modifications);
+            $modifications['update'] = $this->columnFamilyObject($update);
+        } elseif (isset($modifications['drop'])) {
+            $modifications['drop'] = true;
+        }
+        return $this->serializer->decodeMessage(
+            new ModifyColumnFamiliesRequest_Modification(),
+            $this->pluckArray([
```

Comment:
Don't think we need this pluckArray either. Just passing `$modifications` should be sufficient.